CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing to CRA Template Truemark
+
+Thank you for your interest in contributing to our project! We appreciate your interest in contributing.
+
+## How to Contribute
```

Comment:
Let's split this into two sections since if they are an employee of Truemark then they will have the ability to just push the changes directly to our project and don't need to fork.